doc/src/_static/default.css_t

```diff
@@ -290,10 +290,12 @@ p.admonition-title:after {
 }
 
 pre {
+    font-family: DejaVu Sans Mono, DejaVu Sans, monospace;
```

Comment:
DejaVu Sans is a non-monospace font, so I think it shouldn't be listed here.

Reply:
Ohh okk okk i will change it .

Reply:
I wrote it just because the docs were good by changing the css to ```DejaVu Sans Mono, DejaVu Sans, monospace```. Now after looking up you are right that it is non-monospace font.
I had changed it.